test/common/tests.go

```diff
@@ -13,6 +13,7 @@ var (
 	HAPPY_PATH_TESTS = []TestCase{
 		// Add all happy path tests to be executed after RHMI installation is completed here
 		{"Verify RHMI Config CRs Successful", TestRHMIConfigCRs},
+		{"Verify auth flow first broker login", TestAuthDelayFirstBrokerLogin},
```

Comment:
btw: this test has been added to integreatly-test-cases repo as "A16 - Custom first broker login flow" (https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases/blob/master/tests/installation/a16-custom-first-broker-login-flow.md)
We probably need to rename it to match, or at least prepend the name with "A16" code.